DirectEmployers/MyJobs
# Switch from redis to kinesis so we can use lambda.

## What's this PR do?
Switches from using redis for the error queues to Kinesis.

## How should this be manually tested?
Add AWS credentials.
Cause errors.
Confirm that errors are getting added to the Kinesis stream.
Check out Kibana too. They should be there.
